gui/browsable/src/TObjectHolder.cxx

```diff
@@ -13,9 +13,26 @@
 
 using namespace ROOT::Experimental::Browsable;
 
+///////////////////////////////////////////////////////////////////////////
+/// Check if object is not registered in some global lists
+/// Prevent double deletion
+
+void TObjectHolder::CheckOwnership(TObject *obj)
```

Comment:
The method is called `CheckOwnership`, but it changes (remove) the ownership, isn't it? If so, should it be called `RemoveOwnership`, or `CancelOwnership`, or something similar?

Reply:
I renamed to `ClearROOTOwnership`

Reply:
And did rebase again